src/gui/README.md

```diff
@@ -75,10 +75,12 @@ To open the GUI from the command-line (this command does not return until the GU
 ```
 gui::show
 gui::show script
+gui::show script interactive
```

Comment:
When would a user want to specify interactive=false?

Reply:
It would mainly be if doing something like a save_image, but with additional commands possibly not captured at the moment (like load_drc, so something).